decidim-system/app/forms/decidim/system/update_organization_form.rb

```diff
@@ -29,6 +29,7 @@ class UpdateOrganizationForm < Form
         [:authentication, String],
         [:enable_starttls_auto, Boolean]
       ]
+      attribute :password, String
```

Comment:
Few lines below this you'll find this:
```ruby
      attr_writer :password
```

Could you remove that?

Also, after that, take a look at the `encrypted_smtp_settings` method. We should also change that accordingly after the above change:
```ruby
      def encrypted_smtp_settings
        smtp_settings["from"] = set_from

        smtp_settings.merge(encrypted_password: Decidim::AttributeEncryptor.encrypt(attributes["password"]))
      end
```

Note the change inside `Decidim::AttributeEncryptor.encrypt(...)`.

Reply:
Thanks, I understand what you want to achieve here. 😄

It also required to update the `def password` method accordingly: https://github.com/decidim/decidim/pull/9925/commits/8204716d215d8db44e120ca604c4059598efc579

Thanks for review!